deploy/helm-operator-deployment.yaml

```diff
@@ -0,0 +1,54 @@
+---
+  apiVersion: apiextensions.k8s.io/v1beta1
+  kind: CustomResourceDefinition
+  metadata:
+    name: fluxhelmresources.integrations.flux
+  spec:
+    group: integrations.flux
+    names:
+      kind: FluxHelmResource
+      listKind: FluxHelmResourceList
+      plural: fluxhelmresources
+    scope: Namespaced
+    version: v1   
```

Comment:
`v1alpha` at this point, I think.

Reply:
:+1:

Reply:
done

Reply:
It should be `v1alpha1` if we want to use Kubernetes version naming.